### DISCUSSION TITLE
Searching for source code in MooseApp

### DISCUSSION THREAD

**Discussion by Emmanuel-324** [idaholab/moose#28856] [Q&A General] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Hello, I have been looking for this source code in my MooseApp(MatDiffusion.C) but cannot find it. I have seen it online on the MOOSE page and source documentation. However, I am unable to locate it in my MOOSE directory. Any help?

**Comment by GiudGiud** [comment]
Hello

It wont be in your app folder, it will be in the moose folder, in moose/src/kernels

Guillaume

**Reply by Emmanuel-324** [reply]
Yes, that was what I thought yet there is no moose/src/kernels folder. What I see is moose/and_other_folders_except_for src or include. And I cannot find which module it is associated with.

**Reply by GiudGiud** [reply, marked as answer]
oh it s in framework/src/kernels

**Reply by Emmanuel-324** [reply]
Thanks! Found it!
